chi_analysis/add_systematics_qbh.py

```diff
@@ -162,17 +271,18 @@
 print plots
         
 for massbin in massbins:            
-    plots[massbins.index(massbin)]=plots[massbins.index(massbin)].Rebin(len(chi_binnings[massbins.index(massbin)])-1,'QCDQBH'+energy+'_6#chi'+str(massbin[0])+'_'+str(massbin[1])+"_rebin1",chi_binnings[massbins.index(massbin)])
+    plots[massbins.index(massbin)]=plots[massbins.index(massbin)].Rebin(len(chi_binnings[massbins.index(massbin)])-1,'qbh_'+energy+'_6_#chi'+str(massbin[0])+'_'+str(massbin[1])+"_rebin1",chi_binnings[massbins.index(massbin)])
```

Comment:
Can you name it QBHADD and QBHRS1?
Then the calculcate limit script will be able to calculate it in one go with all others and the calculate_limit_shape_qbh script can be dropped.